composer.json

```diff
@@ -20,7 +20,8 @@
         "source": "https://github.com/ramsey/uuid"
     },
     "require": {
-        "php": ">=5.3.3"
+        "php": ">=5.3.3",
+        "paragonie/random_compat": "^2.0"
```

Comment:
Actually, it shouldn't really be much of an issue, since most systems will match a CSPRNG in paragonie/random_compat before it gets to the OpenSSL block, so not a big deal.